pkgs/applications/networking/mailreaders/notbit/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, makeWrapper, pkgconfig,
+  gettext, openssl
+}:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  name = "notbit-${version}";
+  version = "2018-01-09";
+
+  src = fetchFromGitHub {
+    owner  = "bpeel";
+    repo   = "notbit";
+    rev    = "8b5d3d2da8ce54abae2536b4d97641d2c798cff3";
+    sha256 = "1623n0lvx42mamvb2vwin5i38hh0nxpxzmkr5188ss2x7m20lmii";
+  };
+
+  nativeBuildInputs = [ autoreconfHook pkgconfig ];
+
+  buildInputs = [ makeWrapper openssl gettext ];
```

Comment:
Is `makeWrapper` invoked via a hook or something? Otherwise, it seems unused (and should probably be treated as native if it is used).